package.json

```diff
@@ -5,8 +5,8 @@
     "packages/*"
   ],
   "scripts": {
-    "build": "lerna run build",
-    "prepublishOnly": "yarn check-snapshots && yarn build && yarn typecheck && yarn lint && yarn format:ci && yarn test",
+    "build": "lerna run prepare",
+    "prepublish": "yarn check-snapshots && yarn build && yarn typecheck && yarn lint && yarn format:ci && yarn test",
```

Comment:
Yeah, it does run on `yarn install`. The reason why prepublish will not work is because it's in the root package file.

I renamed the script to `publish-packages`